app/models/answer.rb

```diff
@@ -4,4 +4,8 @@ class Answer < ActiveRecord::Base
   belongs_to :test
 
   resourcify#rolify
+
+  scope :with_true, -> { where(rigth: 'true' ) }
+  scope :current_student, ->(user_id){ where(answers: {user_id: user_id} ) }
```

Comment:
rename it for_current_student or something like that. hard to read Answer.current_student